src/include/plan_node/projection_plan_node.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+#include "plan_node/abstract_plan_node.h"
+
+namespace terrier::plan_node {
+
+/**
+ * Plan node for projection
+ */
+class ProjectionPlanNode : public AbstractPlanNode {
```

Comment:
I am surprised that `ProjectionPlanNode` does not have any member objects? We should find out what the old code actually did with this plan node.

Reply:
It used to have [member variables](https://github.com/cmu-db/peloton/blob/484d76df9344cb5c153a2c361c5d5018912d4cf4/src/include/planner/projection_plan.h#L87-L94), but all of these are replaced by `OutputSchema`